bundler/lib/bundler/man/bundle.1.ronn

```diff
@@ -107,5 +107,5 @@ and execute it, passing down any extra arguments to it.
 
 These commands are obsolete and should no longer be used:
 
-* `bundle cache(1)`
+* `bundle package(1)`
```

Comment:
We should instead remove this from here, either `bundle cache` or `bundle package` are fine.

Reply:
Yes, you are right.